_std/code/_macros.dm

```diff
@@ -211,3 +211,16 @@ while(FALSE)
 #define area_space_nopower(x) (x.type == /area || x.type == /area/allowGenerate || x.type == /area/allowGenerate/trench) //areas where we will skip searching for shit like APCs and that do not have innate power
 												 //blahhh i think faster than istypes
 
+#define ADD_STATUS_LIMIT(target, group, value)\
```

Comment:
Does this define really need to be here? I feel like this can be declared locally in the stomach.dm file since it's really only used in there.

Reply:
It's in here so that there's a general solution available if other people/projects want to play with non-global status effects caps 🤷. Could move it if needed